src/pages/studyView/queryStudyAnalytics.ts

```diff
@@ -0,0 +1,24 @@
+import * as request from "superagent";
+
+export interface IAnalyticsResponse {
+    totalUses: number;
+    totalUniqueUses: number;
+    timesVisited: number;
+    timesUniqueVisited: number;
+}
+
+export interface IQueryResponse extends request.Response {
+    body: IAnalyticsResponse;
+}
+
+export default async function queryStudyAnalytics(
+    studyIds: string[],
+    apiURL: string
+): Promise<IAnalyticsResponse> {
+    try {
+        const res: IQueryResponse = await request.get(`${apiURL}?studyName=${studyIds[0]}`);
+        return res.body;
+    } catch (error) {
+        throw new Error("400");
```

Comment:
We don't really want to catch this error and throw a new one because any error listeners (sentry.js) will then lose the browser provided error context.   Mobxpromise will reflect the error (isError) and that can then be handled in code (maybe just hide badge)

Reply:
Ok, that catch is removed.